nav2_shim_controller/include/nav2_shim_controller/shim_controller.hpp

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2020 Shrijit Singh
```

Comment:
This package is missing a readme file explaining what it is, how to use it, why you want to use it, parameters in it, etc.